core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -1327,6 +1329,7 @@ LT: '<';
 LTE: '<=';
 GT: '>';
 GTE: '>=';
+DOUBLE_COLON: '::';
```

Comment:
This token declaration isn't needed, as we don't compare the token value in the generated code.